### PR TITLE
Merge dist standalone bootstrap scripts into dist, reuse code

### PR DESCRIPTION
**Pull request by modax**
3 commits, 2 of them have notable changes as written in subject:
```
    Refactor bootstrap bat scripts to use common base
    
    This way all startup scripts contain consistent code. Before they were
    basically the same copy & pasted code with very minor changes (with
    exception of soapui.sh which had slightly more differences). But even
    they could be put into reusable base script.
    
    Base script is called baserunner.bat and it is "call"'ed from each
    bootstrap script. Improvements in comparison to prior code include:
    
    - JRE in SOAPUI_HOME support (1st priority) (previously only in dist_standalone);
    - JAVA_HOME support (2nd priority) (not in prior dist neither dist_standalone);
    - JAVA_OPTS from shell environment is passed to java process as well;
    - groovy.source.encoding=iso-8859-1 is not passed to JAVA_OPTS (used to
      be passed in dist_standalone but not in original dist).
    - bootstrap script no longer modify environment variables globally due
      to use of SETLOCAL
    
    Tested on Windows 7
```
```
    Refactor bootstrap sh scripts to use common base
    
    This way all startup scripts contain consistent code. Before they were
    basically the same copy & pasted code with very minor changes (with
    exception of soapui.sh which had slightly more differences). But even
    they could be put into reusable base script.
    
    Base script is called baserunner.sh and it is sourced from each
    bootstrap script. Improvements in comparison to prior code include:
    
    - JRE in SOAPUI_HOME support (1st priority) (previously only in dist_standalone);
    - JAVA_HOME support (2nd priority) (not in prior dist neither dist_standalone);
    - JAVA_OPTS from shell environment is passed to java process as well;
    - add symlink detection logic from soapui.sh to all *.sh scripts;
    - java process is exec'ed;
    - in a few place more modern shell syntax is used;
    - groovy.source.encoding=iso-8859-1 is not passed to JAVA_OPTS (used to
      be passed in dist_standalone but not in original dist).
    
    Tested on Debian GNU/Linux
```

### CODE REVIEW

File: soapui-installer/src/dist_standalone/bin/installationcomplete.bat [hunk 1]
```diff
@@ -1,2 +0,0 @@
-set CLASSPATH=..\lib\*
-..\jre\bin\java -cp "%CLASSPATH%" TrackOutAppAction InstallSoftware Product SoapUIOS Version %1 success true
```
Comment (IlyaAvdeev) [inline soapui-installer/src/dist_standalone/bin/installationcomplete.bat:2]: changes like removing installationcomplete/uninstallationcomplete/updateinstallation.(bat/sh) won't be applied, please revert it.

**Review by IlyaAvdeev — CHANGES_REQUESTED** [review]

**Comment by IlyaAvdeev** [comment]
please resolve conflicts

File: soapui-installer/src/dist_standalone/bin/installationcomplete.bat (hunk 1 above)
Comment (modax) [reply soapui-installer/src/dist_standalone/bin/installationcomplete.bat:2]: I will. Actually, I touched those by accident :/

Comment (modax) [reply soapui-installer/src/dist_standalone/bin/installationcomplete.bat:2]: No changes to installationcomplete/uninstallationcomplete/updateinstallation anymore except that the ones from `dist_standalone` were removed as they were dupes

**Comment by modax** [comment]
Conflicts resolved

**Comment by ReadyAPIAgent** [comment]
please address the conflicts and recreate the pull request. Thank you.

**Pull request closed without merging**